plugins/blip/basics/components/DashboardSection.js

```diff
@@ -111,7 +120,7 @@ var DashboardSection = React.createClass({
         {titleContainer}
         <div className={containerClass} ref='container'>
           <div className='DashboardSection-content' ref='content'>
-            {this.props.open ? dataDisplay : null}
+            {this.props.togglable ? dataDisplay : null}
```

Comment:
I *think* you might need to change the logic here. We're still trying to check if it's toggled open, right? I'm thinking `this.props.togglable !== togglableState.closed`, since `this.props.togglable` could be defined to `closed` but still display as open, given this logic. Am I thinking about this right?